teacher.rb

```diff
@@ -2,7 +2,7 @@
 
 class Teacher < Person
   def initialize(specialization, age, name = 'unknown', parent_permission: true)
-    super(name, age, parent_permission)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] [OPTIONAL] - It would be a great idea to have an `MIT` license on your profile! When I tried to open your MIT license on your Readme.MD is redirecting me to another page.

Reply:
Hello @Rustamxon7, Neither this comment is optional but I don't really understand what you mean here, can you please tell me more about it? I would like to do it but I am not sure to get what it is.

Reply:
Sure! Every project Microverse students are making must have an MIT license it is a general requirement. For doing this I will share with you a link for this requirement.[Link...](https://github.com/microverseinc/readme-template/blob/master/README.md)  👈🏻 _You can copy and paste and generate it for your project!_

About MIT license:
> The primary terms and conditions of the MIT license are to grant permissions and indemnify developers for future use. Specifically, it grants any person who obtains a copy of the software and associated files the right to use, copy, modify, merge, distribute, publish, sublicense, and sell copies of the software.

Reply:
Thanks, @Rustamxon7